sirepo/auth_db.py

```diff
@@ -324,8 +325,16 @@ class UserRoleInvite(UserDbBase):
 
         @classmethod
         def get_moderation_request_rows(cls):
+            import sirepo.auth
             with sirepo.util.THREAD_LOCK:
-                return cls.search_all_by(status='pending') + cls.search_all_by(status='clarify')
+                t = sirepo.auth.get_module('email').UserModel
```

Comment:
This can be outside of the THREAD_LOCK. When using locks it is good practice to do as little as possible inside the lock to reduce time that other threads may be waiting on the lock

Reply:
6a18786a1c0ea80d306b558a3c9d83ef8d1df2d8